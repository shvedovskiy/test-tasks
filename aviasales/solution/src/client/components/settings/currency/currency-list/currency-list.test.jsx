/* eslint-disable no-undef */
import React from 'react';

import CurrencyList from './currency-list';


describe('<CurrencyList />', () => {
  const props = {
    selectedCurrency: 'Russian Rouble',
    aliases: {
      'Russian Rouble': 'RUB',
      'US Dollar': 'USD',
      'Euro': 'EUR',
    },
    handleChangeCurrency: jest.fn(),
  };
  const currencyList = shallow(<CurrencyList {...props} />);

  it('renders properly', () => {
    expect(currencyList).toMatchSnapshot();
  });
});
