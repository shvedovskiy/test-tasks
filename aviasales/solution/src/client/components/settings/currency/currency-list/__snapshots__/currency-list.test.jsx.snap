// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyList /> renders properly 1`] = `
<styled__CurrencySwitcher>
  <CurrencyButton
    alias="RUB"
    checked={true}
    currencyName="Russian Rouble"
    handleChangeCurrency={[MockFunction]}
    key="Russian Rouble"
  />
  <CurrencyButton
    alias="USD"
    checked={false}
    currencyName="US Dollar"
    handleChangeCurrency={[MockFunction]}
    key="US Dollar"
  />
  <CurrencyButton
    alias="EUR"
    checked={false}
    currencyName="Euro"
    handleChangeCurrency={[MockFunction]}
    key="Euro"
  />
</styled__CurrencySwitcher>
`;
